src/sema/namespace.rs

```diff
@@ -25,6 +25,14 @@ use solang_parser::{
 };
 use std::collections::HashMap;
 
+/// This struct provides context information for the `resolve_type` function.
```

Comment:
```suggestion
/// Provides context information for the `resolve_type` function.
```

It's an enum, but saying this is not necessary anyways